cli/providers/aws/logs/logs.go

```diff
@@ -0,0 +1,31 @@
+package logs
+
+import (
+	"fmt"
+	logger "log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/pchico83/i2kit/cli/schemas/service"
+)
+
+//Logs prints the logs asociatted to a given instance
+func Logs(s *service.Service, startTime *int64, config *aws.Config, log *logger.Logger) error {
+	svc := cloudwatchlogs.New(session.New(), config)
+	input := &cloudwatchlogs.FilterLogEventsInput{
+		LogGroupName: aws.String(fmt.Sprintf("i2kit-%s", s.Name)),
```

Comment:
can we make the i2kit- prefix a constant? and use the same across, so that it's easier to not mess it up.

Reply:
I will do that in a separate PR